### PR TITLE
[improve] add unit test for services endpoint in Cloudera manager API

### PR DESCRIPTION
**Pull request by vladislav-sidorovich**
Increase test coverage

### CODE REVIEW

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    verify(writer, never()).write(any(char[].class));
+    verify(writer, never()).write(any(char[].class), anyInt(), anyInt());
+  }
+
+  private String tojsonl(String json) throws Exception {
+    JsonNode jsonNode = new ObjectMapper().readTree(json);
+    return jsonNode.toString();
+  }
+
+  private String readString(String name) throws IOException, URISyntaxException {
```
Comment (sayuzbas-google) [inline dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java:177]: nit: `readJsonFileAsString`

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    verify(writer, never()).write(anyChar());
+    verify(writer, never()).write(anyString());
+    verify(writer, never()).write(anyString(), anyInt(), anyInt());
+    verify(writer, never()).write(any(char[].class));
+    verify(writer, never()).write(any(char[].class), anyInt(), anyInt());
+  }
+
+  private String tojsonl(String json) throws Exception {
+    JsonNode jsonNode = new ObjectMapper().readTree(json);
+    return jsonNode.toString();
```
Comment (sayuzbas-google) [inline dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java:174]: can be written in one line:
`return new ObjectMapper().readTree(json).toString();`

**Review by sayuzbas-google — APPROVED** [review]

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java (hunk 1 above)
Comment (vladislav-sidorovich) [reply dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java:183]: It's actually any file. So, change to `readFileAsString`

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java (hunk 2 above)
Comment (vladislav-sidorovich) [reply dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java:180]: Done

**Pull request merged**